docs/docs/setting-up-ci-cd.mdx

```diff
@@ -38,35 +38,18 @@ you can make a test run only if the pull request has a certain label (e.g. “NL
 
 ### Validating Data and Stories
 
-Data validation verifies that there are no mistakes or
-major inconsistencies in your domain, NLU data, or conversation data. To validate
-your data, have your CI run this command:
+Data validation verifies that there are no mistakes or major inconsistencies in your domain, NLU 
+data, or conversation data. To validate your data, have your CI run this command:
 
 ```bash
 rasa data validate --fail-on-warnings --max-history <max_history>
```

Comment:
Should we still include a short explanation of which max_history to choose?

Reply:
Sure, how does this sound (to be placed under the command):

If you use the memoization policy with a non-default `max_history` value in your `config.yml` file, provide the same value in the above validator command. Otherwise, provide the default value of `5`.

Reply:
How about 

If you pass a `max_history` value to a Memoization policy in your `config.yml` file, provide the same value in the above validator command. Otherwise, provide the default value of 5.